src/config.js

```diff
@@ -1,4 +1,4 @@
 var config = {};
 
-config.gatewaysServiceUrl = "https://imec.herokuapp.com";
+config.gatewaysServiceUrl = "http://54.89.128.49:5001";//"https://imec.herokuapp.com";
```

Comment:
Here must be a comment: if it's our AWS server - add this info

Reply:
Added It up, Please take a look now